rllib/algorithms/algorithm.py

```diff
@@ -2734,6 +2746,12 @@ def _run_one_evaluation(
         Returns:
             The results dict from the evaluation call.
         """
+
+        if self.config["off_policy_estimation_methods"]:
```

Comment:
`self.config.off_policy_estimation_methods`

Reply:
On my point above:
We should validate in the config, whether the user is trying to kind of do both: off_policy_setimation PLUS configuring some evaluation_duration stuff. This way we can error out and say: For now, you can only do either one of these types of evaluation: off-policy w/ dataset OR via environment rollouts.

Reply:
ack.

Reply:
fixed.

Reply:
If off_policy_estimation is specified and the input is not a dataset we should raise an error.